src/snovault/elasticsearch/searches/parsers.py

```diff
@@ -159,23 +166,45 @@ def get_must_not_match_filters(self, params=None):
             params=params
         )
 
+    def get_inequality_filters(self, params=None):
+        '''
+        Returns params with (lt:|lte:|gt:|gte:) syntax.
+        '''
+        return self.get_filters_by_condition(
+            key_and_value_condition=lambda _, v: bool(INEQUALITY_REGEX.match(v)),
+            params=params
+        )
+
+    def get_non_inequality_filters(self, params=None):
```

Comment:
just a comment, this is a hilarious method name.   Should probably be dont_not_get_non_inequality_filters() :)

Reply:
Yeah, I agree this is horrible. I actually wish there was a better name to distinguish `=` and `!=` operators from `>`, `>=`, `<`, `<=` operators (or even a name for these symbols besides angle brackets?). Relational or comparison operators seem to include all of them. Somehow `!=` is an equality operator and an inequality operator?

So `get_inequality_filters` as a substitute for _get anything with the `gt:, lt:, gte:, lte:` syntax_ definitely a compromise. And `get_non_inequality_filters` is sort of the consequence of matching the symmetry of `get_non_wildcard_filters` and avoiding the ambiguity of `get_equality_filters` which could imply getting filters with equal signs (including e.g. `file_size=lte:3000`, which is specifically what I want to filter out).